types/react-redux/react-redux-tests.tsx

```diff
@@ -357,22 +357,80 @@ namespace TestTOwnPropsInference {
         return { state: 'string' };
     }
 
-    const ConnectedWithoutOwnProps = connect(mapStateToPropsWithoutOwnProps)(OwnPropsComponent);
+    // `OwnProps` can not be inferred, which is good
+    // const ConnectedWithoutOwnProps = connect(mapStateToPropsWithoutOwnProps)(OwnPropsComponent);
+    // Otherwise, this compiles, which is bad.
+    // React.createElement(ConnectedWithoutOwnProps, { anything: 'goes!' });
```

Comment:
If this no longer compiles, that's good, and this test should be safe to delete entirely.